src/renderer/domain/shared-kernel.ts

```diff
@@ -42,10 +42,12 @@ export const enum WalletType {
   SINGLE_PARITY_SIGNER = 'wallet_sps',
   MULTISHARD_PARITY_SIGNER = 'wallet_mps',
   MULTISIG = 'wallet_ms',
+  WALLET_CONNECT = 'wallet_wc',
```

Comment:
shouldn't `wallet_nw` for Nova Wallet be added here in order to show 2 groups in the wallet tab: Nova Wallet and WalletConnect wallets?